pages/linux/pyrit.md

```diff
@@ -0,0 +1,36 @@
+# pyrit
+
+> WPA/WPA2 cracking tool using computational power.
+> More information: <https://github.com/JPaulMora/Pyrit>.
+
+- Display system cracking speed:
+
+`pyrit benchmark`
+
+- List available cores:
+
+`pyrit list_cores`
+
+- Set ESSID:
+
+`pyrit -e {{ESSID}} create_essid`
+
+- Read and analyze packet capture file:
+
+`pyrit -r {{path/to/file}} analyze`
```

Comment:
```suggestion
- [r]ead and analyze a specific packet capture file:

`pyrit -r {{path/to/file.cap|path/to/file.pcap}} analyze`
```

Optional fix: mnemonic can be added.
Optional fix: two extensions can be presented via `|` syntax.